src/plugins/intel_gna/transformations/pwl_approximation.cpp

```diff
@@ -0,0 +1,544 @@
+                                                 lower_bound,
+                                                 upper_bound,
+                                                 allowed_err_pct > 0.015 ? 0.015 : allowed_err_pct,
+                                                 err_pct);
+    if (segments.size() <= 2) {
+        return false;
+    }
+
+    segments.insert(segments.begin(), {
+        0,
+        segments.front().beta,
+        details::are_floats_equal(fmod(exponent, 1.0), 0.0f) ? -std::numeric_limits<double>::infinity() : 0});
+    segments.back().b = segments.back().beta;
+    segments.push_back({0, 0, std::numeric_limits<double>::infinity()});
+    return true;
+}
+
+template<>
+bool pwl_search<ngraph::opset8::Power>(const std::shared_ptr<ngraph::opset8::Power>& node,
+                                       const std::shared_ptr<ngraph::Node>& fake_quantize,
+                                       double allowed_err_pct,
+                                       double& err_pct,
+                                       std::vector<details::Pwl>& segments) {
+    auto constant = std::dynamic_pointer_cast<ngraph::opset8::Constant>(node->get_input_node_shared_ptr(1));
+    double exponent = 0;
+    if (!ngraph_util::get_constant_value(constant, exponent)) {
+        throw std::runtime_error("The unsupported type of element.");
+    }
+
+    return pwl_search_power(node, exponent, 1, 0, fake_quantize, allowed_err_pct, err_pct, segments);
+}
+
+template<>
+bool pwl_search<ngraph::op::PowerIE>(const std::shared_ptr<ngraph::op::PowerIE>& node,
+                                     const std::shared_ptr<ngraph::Node>& fake_quantize,
+                                     double allowed_err_pct,
+                                     double& err_pct,
+                                     std::vector<details::Pwl>& segments) {
+    auto power = std::dynamic_pointer_cast<ngraph::op::PowerIE>(node);
+    return pwl_search_power(node, power->power, power->scale, power->shift, fake_quantize, allowed_err_pct, err_pct, segments);
+}
+
+template<typename T>
+bool transform_to_pwl(
+    const std::shared_ptr<ngraph::Node>& fake_quantize,
+    const std::shared_ptr<T>& node,
+    double allowed_err_pct) {
+    double err_pct = 0;
+    std::vector<details::Pwl> segments;
+    if (!pwl_search<T>(node, fake_quantize, allowed_err_pct, err_pct, segments)) {
+        return false;
+    }
+
+    std::vector<double> m(segments.size() - 1);
+    std::vector<double> b(segments.size() - 1);
+    std::vector<double> alpha(segments.size());
+    for (size_t i = 0; i < segments.size() - 1; i++) {
+        m[i] = segments[i].m;
+        b[i] = segments[i].b;
+        alpha[i] = segments[i].alpha;
+    }
+    alpha[segments.size() - 1] = segments[segments.size() - 1].alpha;
+
+    auto m_constant = std::make_shared<ngraph::opset8::Constant>(ngraph::element::Type_t::f64,
+        ngraph::Shape{segments.size() - 1}, m);
+    m_constant->set_friendly_name("PWL slope");
```

Comment:
I think it is better to keep name of original variable somehow and avoid spaces in name.
`node->get_friendly_name() + "/pwl_slope"` for example

Reply:
done